### PR TITLE
Improve alert docs

### PR DESCRIPTION
**Pull request by stuartmclean**
I recently raised this issue:
https://github.com/grafana/grafana/issues/11849

But I don't think I would have if my note here in the documentation had existed.

Also updated the example responses in the same section.

### CODE REVIEW

File: docs/sources/http_api/alerting.md [hunk 1]
```diff
@@ -50,19 +50,16 @@ Content-Type: application/json
...
-      {
-        "metric": "fire",
-        "tags": null,
-        "value": 5.349999999999999
-      }
-    "newStateDate": "2016-12-25",
+    "evalData": null,
     "executionError": "",
-    "url": "http://grafana.com/dashboard/db/sensors"
+    "dashboardUri": "http://grafana.com/dashboard/db/sensors"
```
Comment (bergquist) [inline docs/sources/http_api/alerting.md]: should be `url`

File: docs/sources/http_api/alerting.md [hunk 2]
```diff
@@ -50,19 +50,16 @@ Content-Type: application/json
     "id": 1,
     "dashboardId": 1,
     "panelId": 1,
+    "panelUId": "ABcdEFghij"
```
Comment (bergquist) [inline docs/sources/http_api/alerting.md]: should be `dashboardUid`

File: docs/sources/http_api/alerting.md [hunk 3]
```diff
@@ -86,18 +83,47 @@ Authorization: Bearer eyJrIjoiT0tTcG1pUlY2RnVKZTFVaDFsNFZXdE9ZWmNrMkZYbk
...
-  "id": 1,
-  "dashboardId": 1,
-  "panelId": 1,
-  "name": "fire place sensor",
-  "message": "Someone is trying to break in through the fire place",
-  "state": "alerting",
-  "newStateDate": "2016-12-25",
-  "executionError": "",
-  "url": "http://grafana.com/dashboard/db/sensors"
+  "Id": 1,
```
Comment (bergquist) [inline docs/sources/http_api/alerting.md]: This seems like a bug from our side. We should not return json with Capital letters and we should not return all those properties to the user. Please revert this change and I will fix

Comment (stuartmclean) [reply docs/sources/http_api/alerting.md]: Ok - I hope you aren't going to remove evalData - or maybe we can at least leave it in for the end point here: #11936 

Is the idea that the individual alerting `/api/alerts/:id` should be the same format as a single item from the array returned by `api/alerts/`?

**Comment by stuartmclean** [comment]
Just noticed that "message" is not returned by the `/api/alerts` endpoint - but I do see it in the `/api/alerts/:id` endpoint - should I raise an issue for that?

**Comment by bergquist** [comment]
Thank you for contributing to Grafana! 
Even more so since its an improvement to the docs

**Pull request merged**